client/src/components/Settings/SettingsUniquenessScoreAddresses/SettingsUniquenessScoreAddresses.tsx

```diff
@@ -41,7 +41,7 @@ const SettingsUniquenessScoreAddresses: FC<SettingsUniquenessScoreAddressesProps
   const showMoreThanOneAddress = addresses.length > 1;
 
   useEffect(() => {
-    if (isFetchingScore || !ref?.current) {
+    if (isFetchingScore || !ref || !ref.current) {
```

Comment:
this is the same logic -> longer version

Reply:
My suggestion -> `if (isFetchingScore || !ref?.current?.innerText)`

Reply:
Done.